packages/swapper/src/types.ts

```diff
@@ -309,6 +310,15 @@ export type TradeQuoteStep = {
     maxStreamingQuantity?: number
   }
   relayTransactionMetadata?: RelayTransactionMetadata
+  // ButterSwap-specific executable transaction metadata
```

Comment:
nitpick: revert, we don't have this comment anywhere else

Reply:
https://github.com/shapeshift/web/pull/9799/commits/8fc34d4780a60578aec5db6f364254a34b61b749